patches/net/minecraft/world/entity/EntityType.java.patch

```diff
@@ -46,6 +46,22 @@
      }
  
      @Nullable
+@@ -760,6 +_,15 @@
+                 mob.yHeadRot = mob.getYRot();
+                 mob.yBodyRot = mob.getYRot();
+                 mob.finalizeSpawn(p_262637_, p_262637_.getCurrentDifficultyAt(mob.blockPosition()), p_262666_, null, p_262687_);
++
++                if (mob.isSpawnCancelled()) {
++                    // Neo: Discord mob, spawn was cancelled
```

Comment:
~~Discord~~ -> Discard

Reply:
whoops, thats a funny typo, should be resolved in latest commit